integration/spark-common-test/src/test/scala/org/apache/carbondata/spark/testsuite/allqueries/TestQueryWithColumnMetCacheAndCacheLevelProperty.scala

```diff
@@ -282,6 +285,48 @@ class TestQueryWithColumnMetCacheAndCacheLevelProperty extends QueryTest with Be
     sql("drop table if exists alter_add_column_min_max")
   }
 
+  test("Test For Cache set but Min/Max exceeds") {
+    sql("DROP TABLE IF EXISTS carbonCache")
+    sql(
+      s"""
+         | CREATE TABLE carbonCache (
+         | name STRING,
+         | age STRING,
+         | desc STRING
+         | )
+         | STORED BY 'carbondata'
+         | TBLPROPERTIES('COLUMN_META_CACHE'='name,desc')
+       """.stripMargin)
+    sql(
+      "INSERT INTO carbonCache values('Manish Nalla','24'," +
+      "'gvsahgvsahjvcsahjgvavacavkjvaskjvsahgsvagkjvkjgvsackjg')")
+    checkAnswer(sql(
+      "SELECT count(*) FROM carbonCache where " +
+      "desc='gvsahgvsahjvcsahjgvavacavkjvaskjvsahgsvagkjvkjgvsackjg'"),
+      Row(1))
+  }
+
+  test("Cache Blocklet Level testing") {
```

Comment:
1. give proper name for the test case
2. Drop the table before and after completion of each test case
3. Add the property for setting max byte count before the start of each test case and in after all set the property back to default